lib/mbta.js

```diff
@@ -1,12 +1,140 @@
-'use strict'
+//Conor Jennings - MBTA Assignment
```

Comment:
Creating one object to hold all the lines with their corresponding stops would be better in this instance, something like this: `const subwayLines = {
  Red: [
    'South Station',
    'Park Street',
    'Kendall',
    'Central',
    'Harvard',
    'Porter',
    'Davis',
    'Alewife'
  ],
  Green: [
    'Government Center',
    'Park Street',
    'Boylston',
    'Arlington',
    'Copley',
    'Hynes',
    'Kenmore'
  ],
  Orange: [
    'North Station',
    'Haymarket',
    'Park Street',
    'State',
    'Downtown Crossing',
    'Chinatown',
    'Back Bay',
    'Forest Hills'
  ]
}`